docs/composition.ipynb

```diff
@@ -0,0 +1,739 @@
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "shape: (1, 3)\n",
+      "┌─────────┬───────────────────┬─────────────────────┐\n",
+      "│ ILOSTAT ┆ Mean Hours Worked ┆ Median Hours Worked │\n",
+      "│ ---     ┆ ---               ┆ ---                 │\n",
+      "│ i64     ┆ f64               ┆ i64                 │\n",
+      "╞═════════╪═══════════════════╪═════════════════════╡\n",
+      "│ 1       ┆ 37.671921         ┆ 29                  │\n",
+      "└─────────┴───────────────────┴─────────────────────┘\n",
+      "<opendp.polars.OnceFrame object at 0x12ea6f740>\n"
+     ]
+    }
+   ],
+   "source": [
+    "# IN SERVER\n",
+    "m_polars = dp.m.make_private_lazyframe(\n",
+    "    input_domain=df_domain, \n",
+    "    input_metric=dp.symmetric_distance(), \n",
+    "    output_measure=dp.max_divergence(T=float), \n",
+    "    lazyframe=proposed_plan, \n",
+    "    global_scale=1.\n",
+    ")\n",
+    "\n",
+    "df_release = m_polars(df).collect().filter(pl.col(\"ILOSTAT\")==1)\n",
+    "print(df_release)\n",
+    "#TODO: more explanation on the different steps here\n",
+    "\n",
+    "m_comp = dp.c.make_sequential_composition(\n",
+    "    m_polars.input_domain,\n",
+    "    m_polars.input_metric,\n",
+    "    m_polars.output_measure,\n",
+    "    2, \n",
+    "    [18.0, 18.0]\n",
+    ")\n",
+    "\n",
+    "qbl_comp = m_comp(df)\n",
+    "once_frame = qbl_comp(m_polars)\n",
+    "print(qbl_comp(m_polars))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parallel Composition"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Parallel composition can be viewed as an alternative to sequential composition since it's another way to calculate a bound on the total privacy cost of multiple queries. \n",
+    "\n",
+    "Parallel composition involves partitioning the input dataset into disjoint partitions and applying a differentially private mechanism on each chunk independent of all other chunks. Assuming each individual is represented once in the dataset, their data will appear in exactly one chunk. If there are $k$ chunks, there will be $k$ runs of the differentially private mechanisms.\n",
+    "\n",
+    "To apply parallel composition, we need to set up our margin or define our query to allow each individual to influence at most 1 partition, or appear at most in 1 chunk. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Noninteractive Parallel Composition"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the following example, each individual is represented just one each year, quarter, and labor status. Moreover, this example also satisfies the properties of noninteractive sequential composition since all the queries are executed simultaneously. "
```

Comment:
```suggestion
    "In the following example, each individual is represented just once for each combination of year, quarter, and labor status. Moreover, this example also satisfies the properties of noninteractive sequential composition since all the queries are executed simultaneously. "
```